packages/NodeNameResolver/NodeNameResolver.php

```diff
@@ -181,6 +186,12 @@ public function isStringName(string $resolvedName, string $desiredName): bool
             return $desiredName === $resolvedName;
         }
 
+        foreach(self::REGEX_WILDCARD_CHARS as $char) {
```

Comment:
move this after `strcasecmp($resolvedName, $desiredName) === 0`

```php
       if (strcasecmp($resolvedName, $desiredName) === 0) {
             return true;
       }
      
       // add here...
      foreach(self::REGEX_WILDCARD_CHARS as $char) {
      }
      
       return false;
```

Reply:
I will create new PR to move this after last resort to avoid unnecessary loop when possible.